nucleus/security/ssl-impl/src/main/java/com/sun/enterprise/security/ssl/impl/SecuritySupportImpl.java

```diff
@@ -453,10 +453,11 @@ private static void initStores(String keyStoreFileName, char[] keyStorePass, Str
                 for(String trustStoreName : additionalTrustStoreFileNames){
                     trustStoresList.add(loadStore(getProperty(KEYSTORE_TYPE_PROP, KeyStore.getDefaultType()), null, trustStoreName, keyStorePass));
                 }
-            } catch (FileNotFoundException fnfe){
-                _logger.fine("Additional keystore file not found "+fnfe.getMessage());
-            } catch (NullPointerException npe){
+            }
+            catch (NullPointerException npe){
```

Comment:
This is a code smell. The NPE should not be allowed to happen, and not "caught" as an exception

Reply:
Use an if statement instead:
if (variable == null) {
 _logger.fine("No additional keystores or truststores requested");
}